.travis.yml

```diff
@@ -1,3 +1,7 @@
 language: ruby
+sudo: false
+cache:
+  bundler: true
+  apt: true
```

Comment:
Hi @jhermsmeier, thanks for the improvements submitted. Just a minor question: Is there a concrete rationale for explicitly enable caching for `apt`here, too?

After digging into documentation I stumbled over that comment:

>As the new infrastructures does not have APT caches set up, we have removed the mention of it (travis-ci/docs-travis-ci-com@bd8b56d) from our docs.

(found [here](https://github.com/travis-ci/travis-ci/issues/5876))

Is there a benefit of enabling it anyway?